.github/workflows/verify-ensurepip-wheels.yml

```diff
@@ -17,6 +17,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
```

Comment:
Why was this removed? We should bump to v4:

```yaml
- uses: actions/setup-python@v4
  with:
    python-version: '3'
```

Reply:
I could either make the change you suggest or use Python from ubuntu-latest that is enough, I chose the latter.

Reply:
I agree that it works, but it loses consistency with all the other workflows that use (rather than build) python, and ties us to the Python update cadence that Ubuntu use. I would argue that it is also more discoverable on the GHA UI, rather than implicitly using Ubuntu's default Python.

A

Reply:
Besides, unlike Ubuntu's Python, setup-python's builds don't have esoteric patches that deviate behaviour in various ways.

Reply:
Fair enough